GameServer/Logic/GameObjects/Champion.cs

```diff
@@ -21,29 +14,22 @@ public class Champion : Unit
 
         public Shop Shop { get; protected set; }
         public InventoryManager Inventory { get; protected set; }
+        public long RespawnTimer { get; private set; } = 0;
+        public float ChampionGoldFromMinions { get; set; } = 0;
+        public RuneCollection RuneList { get; set; }
+        public List<Spell> Spells { get; private set; } = new List<Spell>();
```

Comment:
This should in my opinion be initialized in the object constructor right before filling it.